Scripts/generate_package_manifest.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+set -e
+
+function usage
+{
+  echo "Usage: ${0} -zip-path <path to xcframework zip> -o <output file path> --EE"
+}
+
+while [[ $# -gt 0 ]]
+do
+  key=${1}
+  case $key in
+      -v)
+      VERSION=${2}
+      shift
+      ;;
+      -zip-path)
+      ZIP_PATH=${2}
+      shift
+      ;;
+      --EE)
+      EE=YES
+      ;;
+      -o)
+      OUTPUT_DIR=${2}
+      shift
+      ;;
+      *)
+      usage
+      exit 3
+      ;;
+  esac
+  shift
+done
+
+if [ -z "$EE" ]
+then
+  PRODUCT_NAME="CouchbaseLiteSwift"
+else
+  PRODUCT_NAME="CouchbaseLiteSwiftEE"
```

Comment:
Is this will be referenced by users when they use Swift Package Manager? Does it need to be different b/w CE and EE? Does the binary file need to be CouchbaseLiteSwiftEE.xx?

Reply:
Right. I think we can keep it as same.